core/src/impl/Cabana_CartesianGrid.hpp

```diff
@@ -20,52 +20,62 @@ namespace Cabana
 {
 namespace Impl
 {
-template<typename Scalar>
 class CartesianGrid
 {
   public:
 
-    Scalar _min_x;
-    Scalar _min_y;
-    Scalar _min_z;
-    Scalar _max_x;
-    Scalar _max_y;
-    Scalar _max_z;
-    Scalar _dx;
-    Scalar _dy;
-    Scalar _dz;
-    Scalar _rdx;
-    Scalar _rdy;
-    Scalar _rdz;
+    double _min_x;
```

Comment:
As per my comment on Slack, I don't really like templating on data stype..but not do I like being married down to double (I think it makes this Cartesian Grid less generic, even though it makes sense for the Vertlet being the user)